bin/githooks/msg-validation.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+
+/**
+ * GIT COMMIT-MSG hook for validating commit message to fit schema.
+ */
+
+'use strict'
+
+let fs = require('fs')
+
+const allowedMaxLength = 70
+const allowedPattern = /^(\w*)(\(.*\))?: (.+)$/
```

Comment:
- `\w*` also allows 0 characters. -> `\w+` = at least 1
- if a scope is there, it also shouldn't be empty (no empty brackets -> `.+` instead of `.*`).